src/test/java/com/github/mauricioaniche/ck/MethodInvocationsTest.java

```diff
@@ -0,0 +1,208 @@
+package com.github.mauricioaniche.ck;
+
+import com.google.common.collect.Sets;
+import junit.framework.Assert;
+import org.apache.commons.lang.ArrayUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class MethodInvocationsTest extends BaseTest {
+    private static Map<String, CKClassResult> report;
+
+    @BeforeAll
+    public static void setUp() {
+        report = run(fixturesDir() + "/MethodInvocation");
+    }
+
+    //Test method invocations
+    private void testMethodInvocation(Map<String, Set<String>> methodInvocations, Set<CKMethodResult> methods){
+        for (CKMethodResult methodResult : methods){
+            Assert.assertEquals(
+                    methodInvocations.get(methodResult.getQualifiedMethodName()),
+                    methodResult.getMethodInvocations());
+        }
+    }
+
+    @Test
+    public void methodInvocations1() {
```

Comment:
We indeed have no good test naming in CK... but that really difficult the reviews. It's hard to know what you are precisely testing in each test method. That's ok for now, but in the future, we might need to improve these names..

Reply:
I will rename the tests, I helps a lot with readability and interpreting the test results.